lsp/lsp.go

```diff
@@ -172,13 +184,23 @@ func (s *Server) publishDiagnostics(uri protocol.DocumentUri, diagnostics []prot
 
 func (s *Server) initialize() protocol.InitializeFunc {
 	return func(context *glsp.Context, params *protocol.InitializeParams) (any, error) {
+		s.logger.Infof("Initializing language server\n")
```

Comment:
No `\n`

Reply:
Thanks! I can remove this completely actually.